ansible/roles/passenger/files/etc/nginx/nginx.conf

```diff
@@ -2,6 +2,7 @@ user www-data;
 # Different boxes may have different number of vCPUs, so configure it automatically
 worker_processes auto;
 pid /run/nginx.pid;
+include /etc/nginx/modules-enabled/*.conf;
```

Comment:
This is due to a difference in how the passenger module gets configured between 16 and 20 https://github.com/phusion/passenger/issues/2129. This way works for both 16 and 20, hence no flagging required.